content/en/blog/_posts/2025-xx-xx-jobs-managedby-goes-ga.md

```diff
@@ -0,0 +1,119 @@
+
+## Why delegate Job reconciliation?
+
+Before `managedBy`, every `batch/v1` Job in a cluster was completely “owned” by the built-in Job controller.
+
+That behaviour works well for single-cluster workloads, but it becomes limiting once you introduce multi-cluster
+schedulers and various controllers on top of Jobs.
+
+A few examples:
+- **Multi-cluster orchestration.**  
+  In MultiKueue’s manager-worker architecture, a Job in the management cluster needs its status mirrored from a Job
+  running in a worker cluster. For that to work reliably, we must be able to tell the built-in Job controller to
+  _not reconcile_ certain Jobs.
+- **Custom Job controllers.**  
+  Projects like JobSet, Kubeflow Training Operator, KubeRay, Tekton Pipelines, and AppWrapper provide their own
+  reconciliation logic around Job-like resources and integrate with Kueue and MultiKueue. They need a standard way to
+  say “this controller is responsible for status” without fighting the Job controller.
+- **Operational safety.**  
+  Disabling the Job controller cluster-wide using `--controllers=-job,*` is heavy-handed, often not possible on managed
+  control planes, and doesn’t work well when the same cluster plays both manager *and* worker roles.
+
+The `managedBy` field solves this by letting you delegate Job synchronization on a **per-Job** basis instead of flipping
+a global switch.
+
+## How `spec.managedBy` works
+
+`spec.managedBy` indicates which controller manages this Job.
+
+The semantics are intentionally simple:
+- If `spec.managedBy` is **unset**, or set to the reserved value `kubernetes.io/job-controller`,  
+  the built-in Job controller reconciles the Job as usual.
+- If `spec.managedBy` is set to **any other valid value**, the built-in Job controller **Skips reconciliation** for that Job, 
+  and continues to reconcile all other Jobs in the cluster.
+
+## MultiKueue: the main user today
+
+The first and main consumer of `spec.managedBy` today is **MultiKueue**, the multi-cluster Job dispatcher built on top
+of Kueue.
+
+In a typical MultiKueue setup:
+- You submit a Job to a **management cluster**.
+- MultiKueue decides *where* that Job should actually run and creates a **mirror Job** in a selected worker cluster.
+- The worker Job runs Pods and updates its own status as usual.
+- MultiKueue then **mirrors the status** of the worker Job back into the management-cluster Job.
+
+Here, `spec.managedBy` is the piece that keeps everyone out of each other’s way:
+- On the **management cluster**, the Job has `spec.managedBy` set to a MultiKueue-specific value
+  (for example, `kueue.x-k8s.io/multikueue`).  
+  The built-in Job controller sees that value, skips reconciliation for that Job, and lets MultiKueue be the source of
+  truth for its status.
+- On the **worker cluster**, the mirror Job **does not** set `spec.managedBy`, so it is fully managed by the built-in
+  Job controller.
```

Comment:
This section feels redundant with "## Why delegate Job reconciliation?" so I would propose to skip it.